app/models/performance_report.rb

```diff
@@ -12,15 +12,17 @@ def generate_report
     end
     self.report_content = {
       global: global,
-      zuerich: zuerich
+      zuerich: zuerich,
+      internal_offer: internal_offer
```

Comment:
the internal external thing is covered with the boolean. So either the whole report is internal or it is global